test/test_theme.rb

```diff
@@ -56,7 +56,7 @@ def setup
 
     should "return the resolved path when a symlink & resolved path exists" do
       expected = File.expand_path("./_layouts", @expected_root)
-      assert_equal expected, @theme.send(:path_for, :symlink)
```

Comment:
Not happy with this at all, `Theme#path_for` sounds like it should be an abstraction, looks like it's just a sugary version of `Site#in_theme_dir`